src/components/ui/Panel.jsx

```diff
@@ -83,6 +86,20 @@ export default class Panel extends React.Component {
       window.execOnMapLoaded(() => {
         fire('move_mobile_bottom_ui', height);
       });
+
+      if (height > this.state.defaultHeight) {
```

Comment:
If don't think this is realiable as the Panel can be passed an `initialSize` prop to render it first minimized or maximized. If the initial size is minized and you swipe up, this first test will be considered true, hiding both buttons whereas we are transitioning to default.
Instead of this `defaultHeight` set at first mount, we should set a value when the transtion starts and compare it during the swipe. I think this is exactly what's already done in `holdResizer`, which stores this value in `this.startHeight`, as we need it for the algo controling the Panel resize.